packages/sdk/app-framework/src/plugins/common/layout.ts

```diff
@@ -67,6 +67,8 @@ export const Layout = z.object({
     .string()
     .optional()
     .describe('The identifier of a component to scroll into view when it is mounted.'),
+
+  plankSizing: z.record(z.string(), z.number()).describe('The size of the planks in the layout.'),
```

Comment:
ideally this should be a deck extension of this type since it's pretty deck-specific as opposed to a more general layout component